README.md

```diff
@@ -10,15 +10,58 @@ None
 
 | variable | description | default |
 |----------|-------------|---------|
-
+| `jenkins_slave_user` | Jenkins user | `{{ __jenkins_slave_user }}` |
+| `jenkins_slave_group` | Jenkins group | `{{ __jenkins_slave_group }}` |
+| `jenkins_slave_home` | Jenkins home | `{{ __jenkins_slave_home }}` |
+| `jenkins_slave_package` | package name | `{{ __jenkins_slave_package }}` |
```

Comment:
other undocumented variables:

* `jenkins_slave_ansible_vault_key`
* `jenkins_slave_authorized_keys`

when adding a variable, it is always recommend to define the variable in `defaults/main.yml` to prevent undocumented variables.